src/handler.py

```diff
@@ -27,26 +27,23 @@
     """
 
 
-async def text_handler(update: Update, context: ContextTypes.DEFAULT_TYPE, con: psycopg2.connect) -> None:
-    _ = context
-    if update.message is None:
-        return
-
-    if update.message.from_user is None:
+async def store_message(message: Message, con: psycopg2.connect) -> None:
+    if message.from_user is None:
         logger.warning("Message has no sender. Skipping...")
         return
```

Comment:
This is something that will need to be addressed eventually: messages with no specific sender may occur normally in group chats where messages are sent on behalf of the group itself. One way to address this is to simply add another special user-ID, perhaps a negative one.

You should add a TODO comment or create a new ticket.